RELEASE_PROCESS.md

```diff
@@ -0,0 +1,98 @@
+# Release Process
+
+- [Release Process](#release-process)
+  - [Breaking Changes](#breaking-changes)
+  - [Release Procedure](#release-procedure)
+    - [Creating a new release branch](#creating-a-new-release-branch)
+    - [Cutting a new release](#cutting-a-new-release)
+    - [Tagging Procedure](#tagging-procedure)
+  - [Patch release Procedure](#patch-release-procedure)
+
+This document outlines the release process for Babylon node (babylond)
+
+Babylon follows [semantic versioning](https://semver.org), but with the following deviations to account for state-machine and API breaking changes.
+
+- State-machine breaking changes & API breaking changes will result in an increase of the minor version Y (0.Y.z).
+- All other changes will result in an increase of the patch version Z (0.y.Z).
```

Comment:
So patches are not anymore only for hotfixes?

Reply:
yup, if we have minor non state breaking, non api breaking change we can release it as patch version